src/baseline_bots/utils.py

```diff
@@ -383,7 +383,10 @@ def get_state_value(bot, game, power_name, option="default"):
     firststep_sc = len(game.get_centers(power_name))
     dipnet_comparison = {power: 0 for power in game.map.powers}
     support_count = {power: 0 for power in game.map.powers}
-    for i in range(bot.rollout_length):
+    movement_phase = 0
+    for i in range(3*bot.rollout_length):
+        if game.get_current_phase()[-1]=='M':
```

Comment:
To follow [PEP 8](https://peps.python.org/pep-0008/):

> Use `''.startswith()` and `''.endswith()` instead of string slicing to check for prefixes or suffixes.

```suggestion
        if game.get_current_phase().endswith('M'):
```

Reply:
done!